dcrtimed/dcrtimed.go

```diff
@@ -556,21 +1048,50 @@ func _main() error {
 		}
 		d.httpClient = &http.Client{Transport: tr}
 
-		statusRoute = d.proxyStatus
-		timestampRoute = d.proxyTimestamp
-		verifyRoute = d.proxyVerify
+		statusV1Route = d.proxyStatusV1
+		timestampV1Route = d.proxyTimestampV1
+		verifyV1Route = d.proxyVerifyV1
+
+		statusV2Route = d.proxyStatusV2
+		timestampV2Route = d.proxyTimestampV2
+		timestampsV2Route = d.proxyTimestampsV2
+		verifyV2Route = d.proxyVerifyV2
 	} else {
-		statusRoute = d.status
-		timestampRoute = d.timestamp
-		verifyRoute = d.verify
+		statusV1Route = d.statusV1
+		timestampV1Route = d.timestampV1
+		verifyV1Route = d.verifyV1
+
+		statusV2Route = d.statusV2
+		timestampV2Route = d.timestampV2
+		timestampsV2Route = d.timestampsV2
+		verifyV2Route = d.verifyV2
+	}
+
+	// Top-level route handler
+	d.router.HandleFunc(v2.VersionRoute, d.version).Methods("GET")
+
+	versions, _ := parseAPIVersionsConfig(loadedCfg.APIVersions)
+
+	// Add handlers according to supported API versions in cfg
+	for _, v := range versions {
+		switch v {
+		case v1.APIVersion:
+			// API v1 handlers
+			d.router.HandleFunc(v1.StatusRoute, statusV1Route).Methods("POST")
+			d.router.HandleFunc(v1.TimestampRoute, timestampV1Route).Methods("POST")
+			d.router.HandleFunc(v1.VerifyRoute, verifyV1Route).Methods("POST")
+		case v2.APIVersion:
+			// API v2 handlers
+			d.router.HandleFunc(v2.StatusRoute, statusV2Route).Methods("POST")
+			d.router.HandleFunc(v2.TimestampRoute, timestampV2Route).Methods("POST")
```

Comment:
```suggestion
			d.router.HandleFunc(v2.TimestampRoute, timestampV2Route).Methods("POST", "GET")
```

Allow this endpoint to be called from both POST and GET